src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -38,14 +40,14 @@
   private final boolean overflow;
   private final String territoryName;
   private final IUIContext uiContext;
-  private final static List<String> STATIC_UNITS =
-      Arrays.asList(Constants.UNIT_TYPE_AAGUN, Constants.UNIT_TYPE_FACTORY);// TODO add every "static"
-  // non-movable unit type of every
-  // map.
   private static UnitFlagDrawMode drawUnitNationMode = UnitFlagDrawMode.NEXT_TO;
+  
+  public static final String UNIT_FLAG_DRAW_MODE = "UNIT_FLAG_DRAW_MODE";
```

Comment:
It's an internal string, so if it were an enum value, then:  `UNIT_FLAG_DRAW_MODE.name()` would give you the same string. Generally it's good to avoid `String` for identifier types, otherwise everything becomes a string quite quickly. That is perhaps the one, and compelling reason to have an enum instead of String type.